### PR TITLE
MWPW-137970: make MEP prelinks relative

### PR DESCRIPTION
**Pull request by yesil**
Makes the prelink href relative for personalization fragments.

Resolves: [MWPW-137970](https://jira.corp.adobe.com/browse/MWPW-137970)

**Test URLs:**
- Before: https://main--homepage--adobecom.hlx.page/homepage/index-loggedout
- After: https://main--homepage--adobecom.hlx.page/homepage/index-loggedout?milolibs=mwpw-137970--milo--yesil&mep=%2Fhomepage%2Ffragments%2Fmep%2Fhp.json--target-dc

### CODE REVIEW

File: libs/features/personalization/personalization.js [hunk 1]
```diff
@@ -50,14 +50,21 @@ const DATA_TYPE = {
 };
 
 const createFrag = (el, url, manifestId) => {
-  const a = createTag('a', { href: url }, url);
+  let href = url;
+  try {
+    const { pathname, search, hash } = new URL(url);
```
Comment (yesil) [inline libs/features/personalization/personalization.js:55]: `normalizePath` in the manifest would drop the search and hash in the url.

Comment (chrischrischris) [reply libs/features/personalization/personalization.js:55]: Fragment links don't use hash & search, so we should use normalizePath

**Review by chrischrischris — APPROVED** [review]

**Review by overmyheadandbody — APPROVED** [review]
The 'after' link is still fetching resources from .page, am I missing a testing step?

**Comment by chrischrischris** [comment]
I've updated the after link to show the fixed code path.   Verified that it is now a relative link instead of absolute.

**Comment by yesil** [comment]
@overmyheadandbody tested with a local override, otherwise it is not possible to test on adobe.com directly AFAIK.

**Review by honstar — APPROVED** [review]

**Pull request merged**